AppCenter.podspec

```diff
@@ -44,6 +44,7 @@ Pod::Spec.new do |s|
     ss.tvos.frameworks = 'UIKit'
     ss.vendored_frameworks = "AppCenter-SDK-Apple/AppCenter.xcframework"
     ss.libraries = 'z', 'sqlite3'
+    ss.resource_bundle = { 'AppCenter' => 'PrivacyInfo.xcprivacy' }
```

Comment:
I think we don't need this as we distribute binaries which already include privacy manifest.